api/envoy/config/endpoint/v3/endpoint.proto

```diff
@@ -75,7 +74,10 @@ message ClusterLoadAssignment {
     //    "throttle"_drop = 60%
     //    "lb"_drop = 20%  // 50% of the remaining 'actual' load, which is 40%.
     //    actual_outgoing_load = 20% // remaining after applying all categories.
-    // [#not-implemented-hide:]
+    //
+    // There are issues to support multiple categories at same time. It is decided
```

Comment:
Note that this restriction is not true for gRPC, which has supported drop_overload for a long time.  Please update the comment to reflect that this restriction applies to Envoy, not gRPC.